core/dbt/contracts/graph/parsed.py

```diff
@@ -676,12 +676,17 @@ class Hook(APIObject):
             # the manifest search stuff really requires this, sadly
             'resource_type': {
                 'enum': [NodeType.Source],
+            },
+            'quoting': {
```

Comment:
these members should probably also be `required` and exist under `properties` as a `'type': 'object'`, right? Something like this:

```
'quoting': {
    'type': 'object',
    'additionalProperties': False,
    'properties': {
                'database': {'type': 'boolean'},
                'schema': {'type': 'boolean'},
                'identifier': {'type': 'boolean'},
   },
   'required': ['database', 'schema', 'identifier'],
}
```
```